scripts/monero.py

```diff
@@ -3,12 +3,17 @@
 # https://github.com/cryptnono/cryptnono/pull/38
 
 from pathlib import Path
+import platform
 from os import execl
 
 # Check for the presence of a linux header only present on older kernels e.g.
 # /usr/src/kernels/5.10.225-213.878.amzn2.x86_64/arch/x86/include/asm/fpu/internal.h
 # Note it's not enough to check for the presence of a newer header (api.h)
-fpu_internal = list(Path("/").glob("usr/src/*/arch/*/include/asm/fpu/internal.h"))
+machine = platform.machine()
+if machine == "x86_64":
+    fpu_internal = list(Path("/").glob("usr/src/*/arch/x86/include/asm/fpu/internal.h"))
+else:
+    raise NotImplementedError(f"Architecture {machine} not supported")
```

Comment:
I checked an arm64 Linux system and there's no `fpu/` directory under `/usr/src/linux-aws-headers-6.8.0-1016/arch/arm64/include/asm/` so we can't run the existing bt scripts